transports/dns/src/lib.rs

```diff
@@ -295,20 +295,16 @@ where
                 }) {
                     if dns_lookups == MAX_DNS_LOOKUPS {
                         tracing::debug!(address=%addr, "Too many DNS lookups, dropping unresolved address");
-                        last_err = Some(Error::TooManyLookups);
+                        dial_errors.push(Error::TooManyLookups);   // this is imp
```

Comment:
For the comment, is this meant to say "this is imp", or is there more that suppose to be there?